scipy/signal/_spectral_py.py

```diff
@@ -43,11 +57,19 @@ def lombscargle(x,
     y : array_like
         Measurement values.
     freqs : array_like
-        Angular frequencies for output periodogram.
+        Angular frequencies for output periodogram. Frequencies are normally >= 0.
+        Any peak at -freq will also exist at +freq.
     precenter : bool, optional
-        Pre-center measurement values by subtracting the mean.
-    normalize : bool, optional
-        Compute normalized periodogram.
+        Pre-center measurement values by subtracting the mean, if True. Ignored if
+        `floating_mean` is True.
+    normalize : bool | str, optional
+        Compute normalized or complex (amplitude + phase) periodogram.
+        Valid options are: False/"power", True/"normalize", or "amplitude".
```

Comment:
Another knit-pick: Writing ``` ``False`` ``` instead of False changes the typesetting to Python code in the rendered documentation, making it clear, the Python keyword is meant. I.e, writing
```
 Valid options are: ``False``/``'power'``, ``True``/``'normalize'``, or ``'amplitude'``.
```
produces:

 Valid options are: ``False`` / ``'power'``, ``True`` / ``'normalize'``, or ``'amplitude'``.

Reply:
Thanks. I wasn't sure how to do that. Relatedly, I was having some issues with the rendered docs (the parameter names in the Notes section were being rendered as links to SciPy functions of the same name).

Reply:
Done.

Reply:
Internal SciPy links are denoted by single backticks.